src/analysis/split.jl

```diff
@@ -3,7 +3,7 @@
 
 Split doc string from given expression.
 """
-function split_doc(ex::Expr)
```

Comment:
I don't see how this fix the problem?

Reply:
`ex::Symbol` goes to the case 
```
@case _ 
return nothing, nothing, ex
```

on line 19, instead of throwing a no method error. effectively making `is_function` return false

Reply:
@Roger-luo any news on this one?